Lesson_2_JSON_YAML_XML/Unicorn xml.xml

```diff
@@ -0,0 +1,17 @@
+<root>
+   <name>Uno</name>
+   <age>2</age>
+   <hasOneHorn>true</hasOneHorn>
+   <favoriteFoods>Peaches</favoriteFoods>
```

Comment:
It works like that, although could be also represented as an array as a list of <favoriteFood> items inside of <favoriteFoods> parent element

```
<favoriteFoods>
      <favoriteFood>smth</favoriteFood>
      <favoriteFood>smth else</favoriteFood> 
</favoriteFoods>
```